tests/pylint/runpylint.py

```diff
@@ -17,7 +17,7 @@ def __init__(self):
                                FalsePositive(r"No value for argument 'member_count' in unbound method call$"),
                                FalsePositive(r"No value for argument 'smallest_member_size' in unbound method call$"),
                                FalsePositive(r"Parameters differ from overridden 'do_task' method$"),
-                               FalsePositive(r"Bad option value '(subprocess-popen-preexec-fn|try-except-raise)'"),
+                               FalsePositive(r"Bad option value '(subprocess-popen-preexec-fn|try-except-raise|environment-modify)'"),
```

Comment:
Why is this needed on top of the `pylint: disable` comment?

Reply:
The `environment-modify` is quite new and this prevents older versions of pylint from complaining about unknown/unsupported option being disabled.